borealisscreenrc

```diff
@@ -0,0 +1,27 @@
+chdir $BOREALISPATH
+screen -t "Brian" bash -c "python brian/brian.py"
```

Comment:
-t is screen option to set the title of bash, -c is bash option to execute quoted stuff